superset-frontend/src/components/NavDropdown/index.tsx

```diff
@@ -33,16 +33,16 @@ const NavDropdown = styled(ReactBootstrapNavDropdown)`
 
   :after {
     content: '';
-    height: ${({ theme }) => theme.gridUnit}px;
-    width: ${({ theme }) => theme.gridUnit * 2}px;
-    background: url('/static/assets/images/icons/dropdown-arrow.svg');
```

Comment:
the dropdown-arrow was small. The triangle-down button is bigger, with padding around it. These changes accommodate the layout change.